diff --git a/abseil-cpp/absl/types/optional.h b/abseil-cpp/absl/types/optional.h
index 134b2aff427..99a1fa89b25 100644
--- a/abseil-cpp/absl/types/optional.h
+++ b/abseil-cpp/absl/types/optional.h
@@ -276,7 +276,14 @@ class optional : private optional_internal::optional_data<T>,
   }
 
   // Copy assignment operator, standard semantics
-  optional& operator=(const optional& src) = default;
+  optional& operator=(const optional& src) {
+    if (src) {
+      this->assign(*src);
+    } else {
+      this->destruct();
+    }
+    return *this;
+  }
 
   // Move assignment operator, standard semantics
   optional& operator=(optional&& src) = default;
