diff --git a/abseil-cpp/absl/strings/internal/str_split_internal.h b/abseil-cpp/absl/strings/internal/str_split_internal.h
index 35edf3aa439..9422057eee1 100644
--- a/abseil-cpp/absl/strings/internal/str_split_internal.h
+++ b/abseil-cpp/absl/strings/internal/str_split_internal.h
@@ -352,7 +352,7 @@ class Splitter {
           ar[index].size = it->size();
           ++it;
         } while (++index != ar.size() && !it.at_end());
-        v.insert(v.end(), ar.begin(), ar.begin() + index);
+        v.insert(v.end(), ar.begin(), ar.begin() + ((ptrdiff_t)index));
       }
       return v;
     }
