diff --git a/rtc_base/ip_address_unittest.cc b/rtc_base/ip_address_unittest.cc
index 9ca05c95fe..1e44f659c9 100644
--- a/rtc_base/ip_address_unittest.cc
+++ b/rtc_base/ip_address_unittest.cc
@@ -61,10 +61,14 @@ static const std::string kIPv6PublicAddrAnonymizedString =
     "2401:fa00:4:x:x:x:x:x";
 static const std::string kIPv6PublicAddr2AnonymizedString =
     "2401:0:0:x:x:x:x:x";
-static const std::string kIPv4MappedAnyAddrString = "::ffff:0:0";
-static const std::string kIPv4MappedRFC1918AddrString = "::ffff:c0a8:701";
-static const std::string kIPv4MappedLoopbackAddrString = "::ffff:7f00:1";
-static const std::string kIPv4MappedPublicAddrString = "::ffff:102:0304";
+static const std::string kIPv4MappedAnyAddrString __attribute__((unused)) =
+    "::ffff:0:0";
+static const std::string kIPv4MappedRFC1918AddrString __attribute__((unused)) =
+    "::ffff:c0a8:701";
+static const std::string kIPv4MappedLoopbackAddrString __attribute__((unused)) =
+    "::ffff:7f00:1";
+static const std::string kIPv4MappedPublicAddrString __attribute__((unused)) =
+    "::ffff:102:0304";
 static const std::string kIPv4MappedV4StyleAddrString = "::ffff:192.168.7.1";
 
 static const std::string kIPv4BrokenString1 = "192.168.7.";
