diff --git a/stats/rtc_stats_unittest.cc b/stats/rtc_stats_unittest.cc
index 8a75b78295..3e4277c6aa 100644
--- a/stats/rtc_stats_unittest.cc
+++ b/stats/rtc_stats_unittest.cc
@@ -495,7 +495,7 @@ TEST(RTCStatsTest, SetTimestamp) {
 TEST(RTCStatsDeathTest, ValueOfUndefinedMember) {
   RTCTestStats stats("testId", Timestamp::Micros(0));
   EXPECT_FALSE(stats.m_int32.has_value());
-  EXPECT_DEATH(*stats.m_int32, "");
+  EXPECT_DEATH(static_cast<void>(*stats.m_int32), "");
 }
 
 TEST(RTCStatsDeathTest, InvalidCasting) {
